ipfx/aibs_data_set.py

```diff
@@ -21,7 +21,14 @@ def __init__(self, sweep_info=None, nwb_file=None, h5_file=None,
             self.notebook = lab_notebook_reader.create_lab_notebook_reader(nwb_file, h5_file)
             sweep_info = self.extract_sweep_meta_data()
 
-        self.sweep_table = pd.DataFrame.from_records(sweep_info)
+        self.build_sweep_table(sweep_info)
+
+    def build_sweep_table(self,sweep_meta_data):
+
+        if sweep_meta_data:
+            self.sweep_table = pd.DataFrame.from_records(sweep_meta_data)
+        else:
+            self.sweep_table = pd.DataFrame(columns=self.COLUMN_NAMES)
```

Comment:
AibsDataSet.build_sweep_table should be in EphysDataSet as HBGDataSet needs the same fix.